tools/build_wheels.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e -x
 
-pybindirs="/opt/python/cp35*/bin /opt/python/cp36*/bin /opt/python/cp37*/bin /opt/python/cp38*/bin"
+pybindirs="$*"
```

Comment:
Any particular reason not to keep the python version here ?

Reply:
Yes, I forgot first to change it 🙄 . What I mean is that it's easier to maintain it if the lists of Pythons are in as few places as possible, i.e. Travis, Tox and docs.

Reply:
Agree.

With this in mind, we could also move the list of python version out of tox and define them only in travis. But wecan keep it for another PR.

Reply:
I don't think it would work but if you find a way to overcome those concerns, that would be cool:

1. because I don't think tox/tox.ini can be parametrized in that way
2. I need tox to run locally without Travis involvment